src/components/search-toggle/_search-toggle.scss

```diff
@@ -167,10 +167,10 @@
   }
 }
 
-.header--secondary.horizontal .search-button,
-.header--secondary.mega .search-button,
-.header--tertiary.horizontal .search-button,
-.header--tertiary.mega .search-button {
+.iowa-bar.iowa-bar--full.horizontal .search-button,
+.iowa-bar.iowa-bar--full.mega .search-button,
+.iowa-bar.iowa-bar--narrow.horizontal .search-button,
+.iowa-bar.iowa-bar--narrow.mega .search-button {
```

Comment:
It seems like this could just be reduced to:
```css
.iowa-bar.horizontal .search-button,
.iowa-bar.mega .search-button {
```

Since `.iowa-bar--full` and `iowa-bar--narrow` will be added in every case.

Reply:
Thanks.  Yes, planning on refactoring this more in another pass soon.